Extensions/CMake/SlicerBlockAdditionalLauncherSettings.cmake

```diff
@@ -19,18 +19,33 @@ if(NOT TARGET ConfigureAdditionalLauncherSettings)
   list(INSERT SEM_LAUNCH_COMMAND ${launch_index} ${Slicer_ADDITIONAL_LAUNCHER_SETTINGS})
 
   # Configure script
+
+  # Third party libraries may live at the superbuild top level, add them to
+  # the library paths and paths lists and adjust the number of paths size,
+  # otherwise just use the two local paths
+  if (DEFINED EXTENSION_SUPERBUILD_BINARY_DIR)
+    set(THIRD_PARTY_LIBRARYPATHS "3\\\\path=${EXTENSION_SUPERBUILD_BINARY_DIR}/${Slicer_THIRDPARTY_LIB_DIR}/\${CMAKE_CFG_INTDIR}
+size=3")
+    set(THIRD_PARTY_BINPATHS "3\\\\path=${EXTENSION_SUPERBUILD_BINARY_DIR}/${Slicer_THIRDPARTY_BIN_DIR}/\${CMAKE_CFG_INTDIR}
+size=3")
+  else()
+    set(THIRD_PARTY_LIBRARYPATHS "size=2")
+    set(THIRD_PARTY_BINPATHS "size=2")
+  endif()
```

Comment:
To keep things simpler, I would suggest to systematically add:

```
3\\\\path=${EXTENSION_SUPERBUILD_BINARY_DIR}/${Slicer_THIRDPARTY_BIN_DIR}/\${CMAKE_CFG_INTDIR}
```

to the list.


Here are the cases of expansion:

* Regular extension, single-config: `/path/to/ext-build/bin/.`
* Regular extension, multi-config: `/path/to/ext-build/bin/Release`
* Superbuild extension, single-config: `/path/to/ext-build/bin/.`
* Superbuild extension, multi-config: `/path/to/ext-build/bin/Release`

where `EXTENSION_SUPERBUILD_BINARY_DIR` corresponds to `/path/to/ext-build`